### ISSUE TITLE
Wrong name match for /ws/guid/batch?q=melaleuca

### ISSUE THREAD

**Issue by nickdos** [AtlasOfLivingAustralia/bie-index#105]
http://bie.ala.org.au/ws/guid/batch?q=melaleuca

first hit is wrong - second hit is right.

**Comment by charvolant** [comment]
I get back

```
{

    "melaleuca": [
        {
            "identifier": "http://id.biodiversity.org.au/node/apni/3676360",
            "name": "Melaleuca",
            "acceptedIdentifier": "http://id.biodiversity.org.au/node/apni/3676360",
            "acceptedName": null
        },
        {
            "identifier": "urn:lsid:biodiversity.org.au:afd.taxon:9dd1ef37-d1dc-4d57-9e72-929065af9b87",
            "name": "Oxyops vitiosus",
            "acceptedIdentifier": "urn:lsid:biodiversity.org.au:afd.taxon:9dd1ef37-d1dc-4d57-9e72-929065af9b87",
            "acceptedName": null
        }
    ]

}
```

which looks right?

**Comment by charvolant** [comment]
*Oxyops vitiosus* has a common name of *Melaleuca Snout Beetle* which is probably why it's showing.

**Comment by nickdos** [comment]
I just clicked that link and i get:

```
{  
  "melaleuca":[  
    {  
      "identifier":"urn:lsid:biodiversity.org.au:afd.taxon:9dd1ef37-d1dc-4d57-9e72-929065af9b87",
      "name":"Oxyops vitiosus",
      "acceptedIdentifier":"urn:lsid:biodiversity.org.au:afd.taxon:9dd1ef37-d1dc-4d57-9e72-929065af9b87",
      "acceptedName":"Oxyops vitiosus"
    },
    {  
      "identifier":"http://id.biodiversity.org.au/node/apni/3676360",
      "name":"Melaleuca",
      "acceptedIdentifier":"http://id.biodiversity.org.au/node/apni/3676360",
      "acceptedName":"Melaleuca"
    }
  ]
}
```
and this AVH search shows the same thing:

http://avh.ala.org.au/occurrences/search?taxa=melaleuca

**Comment by nickdos** [comment]
I've made a change to the SOLR params that that service uses but I've not committed yet - mainly because I need some regression test queries to reassure myself I haven't done more harm than good....

**Comment by nickdos** [comment]
Fixed in version 1.2.2
